### ISSUE TITLE
API : /api/monitorings/config introuvable

### ISSUE THREAD

**Issue by DonovanMaillard** [PnX-SI/gn_module_monitoring#126]
Bonjour, 

Sur GeoNature 2.8.1, une installation du module monitoring et un sous-module se fait sans trop de soucis (en supprimant le trigger mentionné dans #121 ). Mais à l'utilisation, le module monitoring ne trouve jamais ses sous-modules, l'interface reste grisée et le spinner tourne en continu. 

Une erreur 404 est renvoyée par geonature/api/monitorings/config

**Comment by Splendens** [comment]
Ah oui, même problème de mon côté. 
Tu as trouvé une solution ?

**Comment by DonovanMaillard** [comment]
non, pas encore cherché pour être honnête ;) on est restés en 2.7.5 pour l'instant

**Comment by joelclems** [comment]
Est ce que vous avez plus de details sur les logs.
La table gn_comons.t_modules contient bien les sous-modules?
Le table gn_monitoring.t_module_complement aussi?
